dataplane/server.go

```diff
@@ -126,57 +111,6 @@ func (d *Dataplane) Stop(ctx context.Context) error {
 	return nil
 }
 
-// InsertRoute inserts a route into the dataplane.
-func (d *Dataplane) InsertRoute(ctx context.Context, route *dpb.InsertRouteRequest) (*dpb.InsertRouteResponse, error) {
-	if len(route.NextHops) == 0 {
```

Comment:
Keep this check in dataplane/handlers/routes.go to avoid an index error when engine.AddIPRoute is called.